include/AutopinPlus/Monitor/PageMigrate/Main.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * This file is part of Autopin+.
+ * Copyright (C) 2015 Technische Universität München - LRR
+ *
+ * This file is licensed under the GNU General Public License Version 3
+ */
+
+#pragma once
+
+#include <AutopinPlus/PerformanceMonitor.h>
+#include <map>
+
+namespace AutopinPlus {
+namespace Monitor {
+namespace PageMigrate {
+
+/*!
+ * \brief Random performance counter
```

Comment:
Random? I guess you forgot to update the comments in this file. Could you please take a closer look at all the files in this